app/src/main/assets/shared_error_style.css

```
@@ -64,17 +64,31 @@ p {
 
 button {
   display: block;
-  height: 36px;
+  height: 48px;
```

If we want to make it clear in the code that the visible height is 36px, we could change the box sizing to be the standard element sizing.
```suggestion
  box-sizing: content-box;
  height: 36px; /* Padding gets added to this value to create actual height */
  padding: 6px 0px;
```